src/shogun/classifier/LDA.cpp

```diff
@@ -46,6 +46,8 @@ void CLDA::init()
 	m_method=AUTO_LDA;
 	m_gamma=0;
 	m_bdc_svd = true;
+  set_compute_bias(false);
```

Comment:
lets do this in a spearate PR ok?

Reply:
actually it was already false we are taking it out of the constructor

Reply:
i see, thx!